src/Moq/Protected/ProtectedMock.cs

```diff
@@ -154,18 +154,20 @@ public ISetupSequentialResult<TResult> SetupSequence<TResult>(string methodOrPro
 
 		#region Verify
 
-		public void Verify(string methodName, Times times, object[] args)
+		public void Verify(string methodName, Times times, object[] args) { Verify(methodName, times, false, args); }
```

Comment:
Nitpick: For consistency with other code, could you please reformat this to:

```diff
-public void Verify(string methodName, Times times, object[] args) { Verify(methodName, times, false, args); }
+public void Verify(string methodName, Times times, object[] args)
+{
+    this.Verify(methodName, times, false, args);
+}
+
```

Reply:
Done